### DISCUSSION TITLE
auto import twin.macro when typing 'tw' in VsCode

### DISCUSSION THREAD

**Discussion by luknl** [ben-rogerson/twin.macro#334] [Ideas]
Hey, I'm looking for a way to auto import `import 'twin.macro'` on the file imports, when you type `tw` (in `<div tw..` for ex), and then press "tab". (VsCode)
Would that be possible?
Thanks!

**Comment by ben-rogerson** [comment]
@luknl I've just published a plugin that'll sidestep this by automatically adding the import - [babel-plugin-twin](https://github.com/ben-rogerson/babel-plugin-twin)

**Reply by luknl** [reply]
Ok awesome, even better, thank you so much!
Do you plan to update the readme with this?

**Reply by ben-rogerson** [reply]
No worries!
Should be in the documentation soon 👍

**Comment by ben-rogerson** [comment]
To answer your question though - take a look at [my vscode snippets](https://gist.github.com/ben-rogerson/c6b62508e63b3e3146350f685df2ddc9) for some shortcuts.
